.github/workflows/build-and-push-head-images.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and push head branch container images
```

Comment:
Can you make it shorter, e.g. `Dockerhub` or something?
The name will appear on the sidebar of https://github.com/metabase/metabase/actions and if it's too long, it'll be cut off.

Reply:
yup, done  :)